CMakeLists.txt

```diff
@@ -423,26 +423,26 @@ endif(WITH_BENCHMARKS)
 
 # Walberla 
 if (WITH_WALBERLA)
-  include(AddExternal)
-  include(Download)
-  download(
-      libs/walberla
-      GIT_REPOSITORY https://i10git.cs.fau.de/walberla/walberla
-      GIT_TAG master)
-  add_external(walberla
-      libs/walberla/src
-      WALBERLA_BUILD_TESTS=off
-      WALBERLA_BUILD_BENCHMARKS=off
-      WALBERLA_BUILD_TOOLS=off
-      WALBERLA_BUILD_TUTORIALS=off
-      WALBERLA_BUILD_SHOWCASES=off
-      WALBERLA_BUILD_DOC=off
-      CMAKE_POSITION_INDEPENDENT_CODE=on
-      )
+  cmake_minimum_required(VERSION 3.11)
+  include(FetchContent)
+  FetchContent_Declare(
+    walberla
+    GIT_REPOSITORY https://i10git.cs.fau.de/walberla/walberla
+    GIT_TAG        master
```

Comment:
Shouldn't you pull a fixed commit that is compatible with the patch? Pulling a tag can fall over on its own...

Reply:
Resolved.